templates/module/Source/module/Private/Generated/OLink/olinkclient.cpp.tpl

```diff
@@ -89,11 +109,11 @@ void {{$Class}}::Broadcast{{Camel .Name}}Changed_Implementation({{ueParam "In" .
 
 void {{$Class}}::Set{{Camel .Name}}_Implementation({{ueParam "In" .}})
 {
-	if (!m_node)
+	if (!m_sink->m_node)
 	{
 		return;
 	}
-	m_node->setRemoteProperty("{{$ifaceId}}/{{.Name}}", {{ueVar "In" .}});
+	m_sink->m_node->setRemoteProperty("{{$ifaceId}}/{{.Name}}", {{ueVar "In" .}});
```

Comment:
getNode could be public function instead of using private field and mark class friend